service/frontend/handler.go

```diff
@@ -391,6 +402,9 @@ func (wh *WorkflowHandler) RecordActivityTaskHeartbeat(
 	sw := wh.startRequestProfile(scope)
 	defer sw.Stop()
 
+	// Count the request in the RPS, but we still accept it even if RPS is exceeded
```

Comment:
I'm assuming the point here is that we want to give preference to some types of requests and not throttle them? Could that cause issues where service is at max capacity and we continue to send some of these requests? For eg. what if clients send a flurry of non-throttled API requests, there will be no throttling at all.